requirements.txt

```diff
@@ -12,7 +12,7 @@ pycryptodomex==3.17
 PyJWT==2.6.0
 pyOpenSSL==23.0.0
 pytz==2022.7.1
```

Comment:
I think we should remove indirect dependencies from this file and it looks like there are quite a few of them

Reply:
Removing indirect dependencies causes some issues. Firstly, the current list of packages was already approved by the legal and changing it requires the legal team to approve it again ([check fossa scan after cleaning requirements.txt](https://drive.google.com/file/d/1DPa6-BUYwTy3YmpzEToWsgIT0v8ARdbj/view)), which will block my changes for some time. Secondly, removing indirect dependencies from requirements.txt prevents them from being scanned by fossa and I am not ok with pushing wheel files with updated indirect dependencies, which may introduce new security vulnerabilities. Because of that, I prefer to leave it as it is as it doesn't cause any issues besides making requirements.txt a bit longer.

Reply:
**UPDATE**: The problems with invalid requirements.txt detected by prodsec-scan is caused by the current problems with fossa so they can be ignored, but I am still against introducing new versions of indirect dependencies without having them scanned.

Reply:
If that's the case then I'm not going to push the issue